core/chain_manager.go

```diff
@@ -537,7 +517,7 @@ func (self *ChainManager) InsertChain(chain types.Blocks) (int, error) {
 			}
 
 			block.Td = new(big.Int)
-			// Do not penelise on future block. We'll need a block queue eventually that will queue
+			// Do not penalize on future block. We'll need a block queue eventually that will queue
```

Comment:
Please correct properly. Do not favour American English over **correct** English ;-) (i.e. penalise)